components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/SettingsApiServiceImpl.java

```diff
@@ -53,4 +56,22 @@ public Response settingsGet(MessageContext messageContext) {
         }
         return null;
     }
+
+    @Override
+    public Response settingsAttributesGet(String ifNoneMatch, MessageContext messageContext) {
+        return Response.ok().entity(getAllApplicationAttributes()).build();
+    }
+
+    private JSONArray getAllApplicationAttributes() {
+        String username = RestApiUtil.getLoggedInUsername();
+        try {
+            APIConsumer apiConsumer = APIManagerFactory.getInstance().getAPIConsumer(username);
+            String tenantDomain = RestApiUtil.getLoggedInUserTenantDomain();
+            return apiConsumer.getAppAttributesFromConfig(tenantDomain);
```

Comment:
Should we convert to respective DTO before responding? We should not return the model object itself as response